app/code/Stanislavz/Chat/view/frontend/templates/chat/chat-form.phtml

```diff
@@ -0,0 +1,48 @@
+<?php /** @var \Stanislavz\Chat\Block\ChatForm $block */ ?>
+<div class="chat-wrapper"
+     id="chat-wrapper"
+     data-mage-init='{"chatModal":{}}'
+     style="display: none"
+>
+    <div id="message-wrapper">
+        <ul class="message-list"></ul>
+    </div>
+    <form action="javascript:void(0)"
+          method="post"
+          class="chat-form"
+          id="chat-form"
+          data-mage-init='{
+            "validate": {},
+            "chatFormWidget":{
+                "action": "<?= $block->escapeUrl($block->getFormAction()) ?>"
+            }
+          }'
+    >
+        <div class="flex-wrap">
+            <div class="field name required">
+                <div class="control">
+                    <textarea name="message"
+                              class="textarea required text"
+                              id="customer_message"
```

Comment:
@ahrrhy Use dash (`-`), not underscore (`_`) as per CSS id/class naming style and CSS spec recommendation